image/xiaomi_depthmap.ksy

```diff
@@ -0,0 +1,95 @@
+meta:
+  id: xiaomi_depthmap
+  endian: le
+  title: Xiaomi depth images
+  license: MIT
+doc: |
+  Parser for Depthmaps embedded in photos taken with last models of Xiaomi phones.
+  Still WIP.
+  See https://github.com/VincenzoLaSpesa/XiaomiDepthMapExtractor for more info.
+seq:
+  - id: magic
+    contents: "PMPD"
+  - id: image_info
+    type: info
+  - id: padding
+    type: padding80  
+  - id: depthmap_info
+    type: type_depthmap_info
+  - id: confidence_map
+    doc: the confidence map, probably
+    type: sector
+    repeat: until
+    repeat-until: not _.has_next
+  - id: depthmap
+    doc: the depthmap
+    size-eos: true
+    
+types:
+  info:
+    doc: Header of the file. Contains the raw lenght and many fields i still don't undestand
+    seq:
+      - id: unknown1
+        type: u2
+      - id: unknown2
+        type: u2
+      - id: unknown3
+        type: u2
+      - id: unknown4
+        type: u2
+      - id: unknown5
+        type: u2
+      - id: unknown6
+        type: u2
+      - id: depth_size
+        type: u2
+      - id: unknown7
+        type: u2
+      - id: padding1
+        size: 4
+      - id: unknown8
+        type: u4
+      - id: unknown9
+        type: u4
+  
+  padding80:
+    doc: 1kb of 0x80
+    seq:
+    - id: head
+      contents: [0x80]
+    - id: padding 
+      size: 1022
+    - id: tail
+      contents: [0x80]
+
+
+  type_depthmap_info: 
+    doc: 17+1007 = 1kb, info about the shape of the depthmap
+    seq:
+    - id: head
+      contents: [01,00,0xFF,0xFF]
```

Comment:
```suggestion
      contents: [0x01, 0x00, 0xff, 0xff]
```

BTW, be careful with prefixing the decimal integer literals with zero in YAML (but also in other languages). This changes the number's interpretation from decimal (base 10) to octal (base 8), and that might cause unexpected results (e.g. `027` translates to `23`). See https://github.com/kaitai-io/kaitai_struct_formats/pull/299#discussion_r471781903 for more details.